frontend/packages/operator-lifecycle-manager/src/utils/useClusterServiceVersions.tsx

```diff
@@ -42,7 +42,7 @@ const normalizeClusterServiceVersions = (
   t: TFunction,
 ): CatalogItem[] => {
   const formatTileDescription = (csvDescription: string): string =>
-    t('operator-lifecycle-manager~## Operator Description\n{{csvDescription}}', { csvDescription });
+    `## ${t('olm~Operator description')}\n${csvDescription}`;
```

Comment:
@yapei This line fixes your comment [here](https://github.com/openshift/console/pull/7496/files?file-filters%5B%5D=.js&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r543106521)